### DISCUSSION TITLE
php8.x mysql naming should be mysqli - shouldn't it?

### DISCUSSION THREAD

**Discussion by kakohari** [oerdnj/deb.sury.org#1861] [Q&A] [answered]
Hi there,

I _think_ that the package should be named according to the php-extension, or do I miss something?

Kind regards
Kai

**Comment by oerdnj** [comment, marked as answer]
Yes, you are missing the fact that package can contain multiple extensions and for each extensions there's `Provides: phpX.Y-<ext>` field.

**Reply by kakohari** [reply]
Yeah, but:
```
php8.1-mysql/bullseye,now 8.1.12-1+0~20221028.28+debian11~1.gbpc35f51 amd64  [Installiert,automatisch]
  MySQL module for PHP
```
might be clearer if read "MySQLi module for PHP"?
People like me might get confused, when php states that mysql-extension is removed (e.g. https://www.php.net/manual/en/function.mysql-connect.php ) and neither the package name nor the apt search function fire the concurrent extension name. So, you're right and this seems to be an issue of apt search. Thanks for your incredibly fast reply!
